packages/common/src/HardhatConfig.js

```diff
@@ -20,14 +20,38 @@ function getHardhatConfig(configOverrides, workingDir = "./") {
   // Solc version defined here so etherscan-verification has access to it
   const solcVersion = "0.8.4";
 
+  // Compilation settings are overridden for large contracts to allow them to compile without going over the bytecode
+  // limit.
+  const LARGE_CONTRACT_COMPILER_SETTINGS = {
+    version: solcVersion,
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 199,
+      },
+    },
+  };
+
   const defaultConfig = {
     solidity: {
-      version: solcVersion,
-      settings: {
-        optimizer: {
-          enabled: true,
-          runs: 199,
+      compilers: [
+        {
+          version: solcVersion,
+          settings: {
+            optimizer: {
+              enabled: true,
+              runs: 1000000,
```

Comment:
Curious why not use 1_000_000 which is more readable?

Reply:
VSCode kept saying it couldn't parse it. I'll try adding it and then running to see if VSCode is just wrong.

Reply:
Ok not a big deal